NEasyAuthMiddleware/Core/EasyAuthAuthenticationHandler.cs

```diff
@@ -84,7 +84,7 @@ protected override Task<AuthenticateResult> HandleAuthenticateAsync()
 
                 _logger.LogTrace($"{nameof(EasyAuthAuthenticationHandler)} found {claims.Count} successful result(s) and mapped them to claims.");
 
-                var identity = new ClaimsIdentity(claims);
+                var identity = new ClaimsIdentity(claims, "EasyAuth");
```

Comment:
This needs to be `NEasyAuthDefaults.AuthenticationSchemeName` as it needs to match the scheme name as described in the 2nd answer of that Stackoverflow link.

Reply:
or better yet use the existing Scheme.Name property for consistency
```
                var identity = new ClaimsIdentity(claims, Scheme.Name);
                var principal = new ClaimsPrincipal(identity);
                var ticket = new AuthenticationTicket(principal, Scheme.Name);
```

Reply:
Changes made as suggested.